src/components/TodoModal/TodoModal.tsx

```diff
@@ -2,11 +1,31 @@
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { getUser } from '../../api';
+import { User } from '../../types/User';
+
+type Props = {
+  todo: Todo;
+  todoCleaner: () => void;
+};
+
+export const TodoModal: React.FC<Props> = ({ todo, todoCleaner }) => {
+  const [user, setUser] = useState<User | null>(null);
```

Comment:
You are initializing your state with props, which can lead to bugs and make the component difficult to reason about. Consider using a different approach to manage your state.